apps/protocol-frontend/src/components/VerifiedCelebrationModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useState } from 'react';
+import { Box, Flex, Stack, Text } from '@chakra-ui/react';
+import { TextList } from './TextList';
+import { useReward } from 'react-rewards';
+import { UIContribution } from '@govrn/ui-types';
+
+export const VerifiedCelebrationModal = () => {
+  const [celebrating, setCelebrating] = useState(true);
+  const { reward } = useReward('rewardId', 'confetti', {
+    colors: ['#86135B', '#5100E4', '#DF1F97', '#F2A5D5'],
+    onAnimationComplete: () => setCelebrating(false),
+  });
+
+  useEffect(() => {
+    if (celebrating) {
+      reward();
+    }
+  }, [reward, celebrating]);
+
+  return (
+    <Stack
+      spacing="4"
+      width="100%"
+      color="white"
+      background="brand.gradientBackgroundModal"
+    >
+      <Flex direction="column" alignItems="center" textAlign="center">
+        <Text paddingBottom={2} fontWeight="bold">
+          Amazing!
+        </Text>
+        <Flex
+          direction="row"
+          minH="8rem"
+          alignItems="center"
+          textAlign="center"
+          wrap="wrap"
+          maxW="md"
+        >
+          <Text>
+            Your attestation has officially{' '}
+            <Text as="span" fontWeight="bold">
+              verified this contribution
+            </Text>{' '}
+            for your DAO.
+          </Text>
+        </Flex>
+        <Box as="span" id="rewardId" />
+      </Flex>
```

Comment:
That's a good idea -- are you thinking that we should include the contribution's name in the single modal as well? I think that's a good call since it's beneficial to the user even though it's a single contribution.

Reply:
https://user-images.githubusercontent.com/9438776/233689821-12b36456-f7ba-4d50-b71e-029d8e725492.mp4

@amrro added for the single, great idea.

Instead of modifying the TextList and having it take in a single item, and instead of forcing the single contribution to be an array, I explicitly am setting the state to a single one and display it. Less DRY but seems safer to be more explicit with it being a single item.

Reply:
@amrro when you get a chance to check this out let me know if this is what you were thinking! thank you!